proto/osmosis/lockup/lock.proto

```diff
@@ -39,8 +39,10 @@ enum LockQueryType {
 }
 
 message QueryCondition {
-  LockQueryType lock_query_type = 1; // type of lock query, ByLockDuration | ByLockTime
-  string denom = 2; // What token denomination are we looking for lockups of
+  // type of lock query, ByLockDuration | ByLockTime
```

Comment:
Nice change, as we found out these don't get into the go docs. Can you also make an issue in this repo to update all of our proto files for this? (Moving comments above the line they affect, rather than in the same line, to the side?)

Reply:
Created one here: #863